triton/tut/storage.rst

```diff
@@ -387,21 +378,18 @@ used as a rudimentary way to see how much I/O load there is.
 
    g) I can't install my own Python/R/etc libraries.
 
-Next steps
-==========
 
-The next tutorial is about :doc:`interactive jobs <interactive>`.
 
-If you are doing anything IO heavy, you might want to read the
-:doc:`advanced storage page <../usage/storage>`.
 
-Optimizing data storage isn't very glamorous, but it's an important
-part of high-performance computing.  You can find some hints on the
-:doc:`profiling <../usage/profiling>` page.
+What's next?
+==========
```

Comment:
```suggestion
============
```
does anyone else see how github actions inserts the CI warnings right here in the document in the right place?  That's pretty neat.